core/chains/evm/config/config_test.go

```diff
@@ -369,109 +311,16 @@ func TestChainScopedConfig_Profiles(t *testing.T) {
 
 func TestChainScopedConfig_HeadTracker(t *testing.T) {
 	t.Parallel()
-	gcfg := configtest.NewTestGeneralConfig(t)
-	cfg := evmtest.NewChainScopedConfig(t, gcfg)
+	cfg := testutils.NewTestChainScopedConfig(t, nil)
 
 	ht := cfg.EVM().HeadTracker()
 	assert.Equal(t, uint32(100), ht.HistoryDepth())
 	assert.Equal(t, uint32(3), ht.MaxBufferSize())
 	assert.Equal(t, time.Second, ht.SamplingInterval())
 }
 
-func Test_chainScopedConfig_Validate(t *testing.T) {
```

Comment:
Is this behavior covered by another test?

Reply:
Honestly, I don't understand what this test covers anymore. If it's the Validate() function of the top level AppConfig, then this is already tested [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/chainlink/config_general_test.go#L79). This seems like it tested for chain-specific behaviour but from what it looks like, no chain-specific validation takes place inside [config/toml/config.go] anymore. (https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/config/toml/config.go)

Reply:
I don't understand what you mean. We cannot remove validation tests arbitrarily, so are you saying that another EVM test already covers these cases?
https://github.com/smartcontractkit/chainlink/blob/develop/core%2Fchains%2Fevm%2Fconfig%2Ftoml%2Fconfig.go#L291
If there is a redundant _core_ test concerning itself with evm details, then that version should be dropped in favor of this one.

Reply:
My point was that the previous validations assumed some chain-specific logi **per id** which isn't actually there anymore. So I just moved the check in [here](https://github.com/smartcontractkit/chainlink/pull/12971/files#diff-f9bd850deb234901c53501494703b719a9525902fca5cac61048d62d5fef802fR13). Let me know if this works.

Reply:
What do you mean that the per chain logic is not there anymore? IIUC this test is initializing defaults per chain ID, then ensuring that certain fields can or cannot be set along with those defaults.

Reply:
Let's take arbitrum test for example. It sets a `BlockHistory` estimator and then calls `Validate()` which doesn't fail because we don't have any underlying check for Arbitrum's estimator type. Same goes for optimism below. I believe this was something that we did in the past, but we don't do it anymore. Now we simply have to call `Validate()` for each default chain id so we can validate the default values are actually ok, which I do in `config_test.go`.

Reply:
So they were already irrelevant, prior to this change, no because of it?

Reply:
Yeah, I believe so. We only have to validate their default configs just as we do for the fallbacks, which I transferred to `config_test.go`